drafts/builder-pattern-tables.md

```diff
@@ -0,0 +1,354 @@
+---
+title: One Table to Rule them All 
```

Comment:
```suggestion
title: One table to rule them all
```

Reply:
Oh, I would actually disagree on this change, capitalising headlines and titles is perfectly fine (news portals like NYT do it too ;)). But you don't have to change it again :D